Sources/UXKit/AppKit/UXView-AppKit.swift

```diff
@@ -68,6 +69,64 @@
       //       maybe?)
       return CGPoint(x: NSMidX(frame), y: NSMidY(frame))
     }
+    
+    var alpha: CGFloat {
+        get {
+            return self.alphaValue
+        }
+        set {
+            self.alphaValue = newValue
+        }
+    }
+    
+    func setSubViews(enabled: Bool) {
```

Comment:
Those should be private, only used by `isUserInteractionEnabled`. Or maybe the imps should just go into the setter/getter of that. Either way works for me, but let's only add API for things required for compatibility.